content/en/blog/2024/humans-of-otel-na-2024.md

```diff
@@ -0,0 +1,522 @@
+---
+title: The Humans OpenTelemetry - KubeCon NA 2024
+linkTitle: Humans of OTel NA 2024
+date: 2024-12-17
```

Comment:
We'll probably want to set this to at least the 18th. I'll let @svrnm decide :)

Reply:
@svrnm I'm okay if this goes out Thursday or Friday. Ideally I'd like to get it out before the holidays.

Reply:
Then let's try to get it out today :-)

Reply:
```suggestion
date: 2024-12-19
```